src/cashflows.js

```diff
@@ -317,7 +317,7 @@ export function Cashflows(intentToken, isIntegration) {
 
 			// Only continue when the checkoutIntentPromise resolves and thus intent has been validated.
 			self._checkoutIntentPromises.push(() =>
-				self._apiRequest('post', 'payment/apple-pay/get-payment-request?token=' + self._intentToken)
+				self._apiRequest('post', 'payment/apple-pay/get-payment-request?paymentDataOrigin=cashflows-clientlib-js&token=' + self._intentToken)
```

Comment:
In the CreatePaymentJob mehtod in the PaymentJobs controller in our backend you've added this parameter as a header (which has my preference), whereas here you are adding it as a query parameter to the request.  Both need to be aligned.'

I'd also instead include this parameter to the 'startPayment' method (line 526). This method is used by all payment methods, not just ApplePay and GooglePay and is called when the payment is started, as opposed to when the embedded frames payment page is loaded. We'll also need to add this parameter to the CreatePaymentFromIntent method in the PaymentIntents controller in the backend.